File: Boats to Save People.js
var numRescueBoats = function(people, limit) {
  let countBoat=0;
      let i = 0;
    let j = people.length-1
    
    people.sort(function(a,b){
        return (a-b) });
   
    while(i<=j){
        if(people[i]+people[j]<=limit){
           countBoat++;
            i++;
            j--;
        }
        else{
            countBoat++;
            j--;
        }
    }
    return countBoat;
};

File: 917-boats-to-save-people/boats-to-save-people.py
class Solution(object):
    def numRescueBoats(self, people, limit):
        people.sort()
        count_boats=0
        i=0
        j=len(people)-1
        while i<=j:
            if people[i]+people[j]<=limit:
                count_boats+=1
                i+=1
                j-=1
            else:
                count_boats+=1
                j-=1
                
        return count_boats
            
            
             

        

       